locale/en/docs/guides/debugging-getting-started.md

```diff
@@ -77,38 +77,10 @@ info on these follows:
 * The latest version can also be installed independently (e.g. `npm install -g node-inspect`)
   and used with `node-inspect myscript.js`.
 
-#### [Chrome DevTools](https://github.com/ChromeDevTools/devtools-frontend) 55+
```

Comment:
This is just a link to a mirror from the Chromium repo which is what's being tested against the Node inspector. If we remove this we need another alternative "blessed" frontend and to run tests against it.

I think we should keep this in the docs for now since the two interconnected parts (v8 and the devtools frontend) are part of the same open source project (chromium) and v8 is a hard dependency of Node. 

As an alternative - we might want to ship this.